src/components/Error/ErrorView.tsx

```diff
@@ -0,0 +1,91 @@
+import { Trans } from '@lingui/macro'
+import ActionButton from 'components/ActionButton'
+import Column from 'components/Column'
+import Expando from 'components/Expando'
+import Row from 'components/Row'
+import { iconHoverCss } from 'css/hover'
+import { AlertTriangle, Icon, LargeIcon, X } from 'icons'
+import { ReactNode, useState } from 'react'
+import styled from 'styled-components/macro'
+import { Color, ThemedText } from 'theme'
+
+const HeaderIcon = styled(LargeIcon)`
+  flex-grow: 1;
+  margin: 2em 0;
+`
+
+interface StatusHeaderProps {
+  icon: Icon
+  iconColor?: Color
+  iconSize?: number
+  children: ReactNode
+}
+
+export function StatusHeader({ icon: Icon, iconColor, iconSize = 2.5, children }: StatusHeaderProps) {
+  return (
+    <>
+      <Column flex style={{ flexGrow: 1 }}>
+        <HeaderIcon icon={Icon} color={iconColor} size={iconSize} />
+        <Column gap={0.75} flex style={{ textAlign: 'center' }}>
+          {children}
+        </Column>
+      </Column>
+    </>
+  )
+}
+
+const ExpandoContent = styled(ThemedText.Code)`
+  margin: 0.5em;
+`
+
+const StyledXButton = styled(X)`
```

Comment:
I feel like i've seen this component declared a couple times across the widget repo now, could we extract it into a single file and re-use via exporting?